### ISSUE TITLE
No notification on windows 10

### ISSUE THREAD

**Issue by kavalun** [deltachat/deltachat-desktop#2668]
- Operating System: Windows 10
- Delta Chat Version: Version 1.26.0 (git: v1.25.0-9-g1897f54e)
Cant get notification and update for new message, if app in tray. New message gets only after open app from tray. All other apps have notification like thunderbird, discord, skype and more

**Comment by gerryfrancis** [comment]
Duplicate of https://github.com/deltachat/deltachat-desktop/issues/2452 .
